### PR TITLE
Vol encrypt requirements - Apple test run

### PR DESCRIPTION
**Pull request by Pearl1594**
NOTE: Not sure if we want trillian to install the dependencies or have these packages installed on MS and KVM hosts as part of installing cloudstack-management / cloudstack-agent? As based on my understanding volume encryption would be enabled by default in ACS - and isn't something that's controlled by a Global setting or property.

### CODE REVIEW

File: Ansible/templates/CentOS-Base.repo.j2 [hunk 1]
```diff
@@ -26,3 +26,11 @@ baseurl={{ os_repo }}/centos/$releasever/updates/$basearch/
 gpgcheck=0
 gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-$releasever
 enabled = {{ update_repo_enabled | default(1) }}
+
+[extras]
+name=CentOS-$releasever - Extras
+mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=extras
```
Comment (rohityadavcloud) [inline Ansible/templates/CentOS-Base.repo.j2:32]: @Pearl1594 do we ned at least this if el7 tries to install haveged and other pkgs?

File: Ansible/roles/kvm/tasks/centos8.yml [hunk 2]
```diff
@@ -218,3 +218,14 @@
...
+- name: Install cryptsetup and rng-tools to support encryption and entropy respectively
+  dnf: name={{ item }} state=present enablerepo=base
+  with_items:
+    - cryptsetup
+    - rng-tools
+  tags:
+    - kvm
+
+- name: Start rng service
+  shell: "systemctl start rngd"
```
Comment (rohityadavcloud) [inline Ansible/roles/kvm/tasks/centos8.yml:231]: Worth checking if this is required by default, or does installing pkg ensure this is already running on kvm hosts?

File: Ansible/roles/cloudstack-manager/tasks/centos8.yml [hunk 3]
```diff
@@ -69,9 +69,15 @@
...
 
-- name: install rng-tools to get entropy
+- name: install haveged to get entropy
   dnf: name=haveged state=present
 
+- name: start haveged for entropy
+  service:
+    name: haveged
+    state: started
+    enabled: yes
```
Comment (rohityadavcloud) [inline Ansible/roles/cloudstack-manager/tasks/centos8.yml:79]: Worth cehcking if simply installing this enable/runs the service?

Comment (Pearl1594) [reply Ansible/roles/cloudstack-manager/tasks/centos8.yml:79]: as mentioned above

File: Ansible/templates/CentOS-Base.repo.j2 (hunk 1 above)
Comment (Pearl1594) [reply Ansible/templates/CentOS-Base.repo.j2:32]: isn't required

**Comment by rohityadavcloud** [comment]
not necessary now - we bundle/install dependency via pkgs now.

**Pull request closed without merging**